eventing/samples/gcp-pubsub-source/README.md

```diff
@@ -20,7 +20,8 @@ source is most useful as a bridge from other GCP services, such as
 
 1. Setup
    [Knative Eventing](https://github.com/knative/docs/tree/master/eventing)
-   using the `release-with-gcppubsub.yaml` file. Start by creating a dummy
+   using the `release-gcppubsub.yaml` file instead of
```

Comment:
FYI -- the earlier file installed all the other sources + the GCP pubsub source, but the new file installs only the GCP pubsub source (leaving out the others). It looks like this tutorial doesn't make use of any other sources, so that should be fine.